packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/MemoryTests.kt

```diff
@@ -82,6 +84,10 @@ class MemoryTests {
         mappedMemorySize = numberOfMemoryMappedBytes(command)
         assertTrue(mappedMemorySize >= 99 * oneMB && mappedMemorySize < 102 * oneMB, "Committing the 100 objects should result in memory mapping ~ 99 MB. Current amount is ${bytesToHumanReadable(mappedMemorySize)}")
 
+        // Change callback are not automatically unregistered if the tokens are release, so for now
```

Comment:
I removed the manual unregistration with https://github.com/realm/realm-kotlin/commit/8deaef1c63fa20b4960bbbbfd866cdb686c93ba1, so not need anymore ( ... and the symbols are also gone 🤪)